packages/cyberstorm/src/components/Layout/Teams/TeamSettings/TeamMembers/TeamMembers.module.css

```diff
@@ -50,3 +50,8 @@
 .kickDescriptionUserName {
   color: var(--color-highlight);
 }
+
+.createAddMemberDialogFooter {
```

Comment:
Is this class actually used anywhere?

Reply:
I miss wrote the class name, should have been `addMemberDialogFooter`